github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRTriggerMode.java

```diff
@@ -6,10 +6,10 @@
  * @author Kanstantsin Shautsou
  */
 public enum GitHubPRTriggerMode {
-    CRON ("Cron with persistence"),
-    HEAVY_HOOKS ("Experimental: Hooks with persistence"),
-    HEAVY_HOOKS_CRON ("NOT SUPPORTED: Hooks with persistence and periodic cron check"),
-    LIGHT_HOOKS("NOT SUPPORTED: Hooks without persistence");
+    CRON ("Cron"),
+    HEAVY_HOOKS ("Hooks"),
+    HEAVY_HOOKS_CRON ("NOT SUPPORTED: Hooks plus Cron"),
+    LIGHT_HOOKS("NOT SUPPORTED: Light Hooks");
```

Comment:
With persistence or not is very critical for understanding imho. It influences on different aspects of how you comparing whether change appeared and how much additional features you may have. 
:+1: for removing `Experimental` prefix because hooks already works for a long time.